Userland/Applications/FileManager/main.cpp

```diff
@@ -993,6 +993,14 @@ ErrorOr<int> run_in_windowed_mode(ByteString const& initial_location, ByteString
     });
     focus_dependent_delete_action->set_enabled(false);
 
+    auto new_window_action = GUI::Action::create("&New Window", { Mod_Ctrl, Key_N }, TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/new-window.png"sv)), [&](GUI::Action const&) {
+        Desktop::Launcher::open(URL::create_with_file_scheme(directory_view->path()));
+    });
+
+    auto close_window_action = GUI::Action::create("&Close Window", { Mod_Ctrl, Key_W }, TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/close-window.png"sv)), [&](GUI::Action const&) {
```

Comment:
What's the difference between that and the quit action? :thinking:
I would rather have `Ctrl-W` as a secondary shortcut for quit